### DISCUSSION TITLE
megatinycore on a Chromebook - device-specs folder issue

### DISCUSSION THREAD

**Discussion by PaulEast** [SpenceKonde/megaTinyCore#902] [Q&A]
Good morning everyone, firstly I'm in awe of this development and have it working perfectly on a Linux box but I'm now trying to get it working on my Chromebook.

I have the board package and additional board manager link in place on Arduino version1.8.13 (installed on the Chromebook linux installation).

When compiling I get 2 error messages:

`Warning: platform.txt from core 'megaTinyCore' contains deprecated compiler.path={runtime.tools.avr-gcc.path}/bin/, automatically converted to compiler.path=/usr/bin/. Consider upgrading this core.
Detecting libraries used...`

and then...

`avr-g++: error: device-specs/specs-attiny404: No such file or directory
exit status 1
Error compiling for board ATtiny3224/1624/1614/1604/824/814/804/424/414/404/214/204.`

I have checked and there is a device-specs/specs-attiny404 folder.

I am at the edge of my knowledge on Linux and the backend of Arduino, if this is just an edge case that won't work that's fine but I thought I'd see if the collective wisdom here could help.

Thanks in advance.

**Comment by SpenceKonde** [comment]
Hmm, @per1234 (cause he's the only one who has a chance in hell of knowing the answer) - what should I be using instead of {runtime.tools.avr-gcc.path} ???

**Reply by PaulEast** [reply]
Thanks for even considering the question, I knew it would be a little bit "out there"!

**Comment by SpenceKonde** [comment]
I'm locking this thread, as this is an issue (and one of the highest severity, and one which requires knowledge I do not possess in order to correct), not a discusssion: There is unambiguously a defect in the core, hence this needs to be tracked as an action item to be completed, not a conversation that can be ignored. 

Further discussion should be in issue #903
